inheritance/README.md

```diff
@@ -0,0 +1,31 @@
+# Inheritance 
```

Comment:
This is a very nice readme. When you want to emphasize certain words, you can use the ticks `Class` to make nice code-blocks for classes.